ex03.ex

```diff
@@ -149,7 +165,16 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
+  def won({a1,a2,a3,a4,a5,a6,a7,a8,a9}) do
```

Comment:
# -3

This is interesting, but it really doesn't seem to be easy to read. Finding a mistake in this would be difficult.

I was looking for something like:

~~~ elixir
  def won({ a, a, a,
            _, _, _,
            _, _, _ }), do: a

  def won({ _, _, _,
            a, a, a,
            _, _, _ }), do: a

  def won({ _, _, _,
            _, _, _,
            a, a, a }), do: a

  def won({ a, _, _,
            a, _, _,
            a, _, _ }), do: a

  def won({ _, a, _,
            _, a, _,
            _, a, _ }), do: a

  def won({ _, _, a,
            _, _, a,
            _, _, a }), do: a

  def won({ a, _, _,
            _, a, _,
            _, _, a }), do: a

  def won({ _, _, a,
            _, a, _,
            a, _, _ }), do: a

  def won(_), do: false
~~~

or

~~~ elixir
  def won1({ a, a, a,  _, _, _,  _, _, _ }), do: a
  def won1({ _, _, _,  a, a, a,  _, _, _ }), do: a
  def won1({ _, _, _,  _, _, _,  a, a, a }), do: a
  def won1({ a, _, _,  a, _, _,  a, _, _ }), do: a
  def won1({ _, a, _,  _, a, _,  _, a, _ }), do: a
  def won1({ _, _, a,  _, _, a,  _, _, a }), do: a
  def won1({ a, _, _,  _, a, _,  _, _, a }), do: a
  def won1({ _, _, a,  _, a, _,  a, _, _ }), do: a
  def won1(_), do: false
~~~